doc/classes/CanvasItem.xml

```diff
@@ -475,32 +476,33 @@
 		<method name="get_transform" qualifiers="const">
 			<return type="Transform2D" />
 			<description>
-				Returns the transform matrix of this item.
+				Returns the transform matrix of this [CanvasItem].
 			</description>
 		</method>
 		<method name="get_viewport_rect" qualifiers="const">
 			<return type="Rect2" />
 			<description>
-				Returns the viewport's boundaries as a [Rect2].
+				Returns this node's viewport boundaries as a [Rect2]. See also [method Node.get_viewport].
 			</description>
 		</method>
 		<method name="get_viewport_transform" qualifiers="const">
 			<return type="Transform2D" />
 			<description>
-				Returns the transform from the coordinate system of the canvas, this item is in, to the [Viewport]s embedders coordinate system.
```

Comment:
Insane that this description has been left like this for so long. I don't even have any idea on what this is trying to say.